src/debugAdapter.ts

```diff
@@ -5,7 +5,7 @@ import path = require('path');
 import net = require('net');
 import { DebugProtocol } from 'vscode-debugprotocol';
 import cp = require('child_process');
-import { NullLogger } from './logging/null';
+import { FileLogger } from './logging/file';
```

Comment:
The logging needs to remain as Null Logger for the time being.  There's no error checking of the log file exists so it blows up the DebugAdapter.  If we really want the file logger at least comment it out

`// import { FileLogger } from './logging/file';`